Makefile

```diff
@@ -208,6 +208,23 @@ batcher_send_halo2_ipa_task_burst_5: batcher/client/target/release/batcher-clien
 		--vk test_files/halo2_ipa/params.bin \
 		--repetitions 5
 
+batcher_send_halo2_kzg_task: batcher/client/target/release/batcher-client
+	@echo "Sending Halo2 KZG 1!=0 task to Batcher..."
+	@cd batcher/client/ && cargo run --release -- \
+		--proving_system Halo2KZG \
+		--proof test_files/halo2_kzg/proof.bin \
+		--public_input test_files/halo2_kzg/pub_input.bin \
+		--vk test_files/halo2_kzg/params.bin \
+
```

Comment:
Missing `proof_generator_addr` flag

Reply:
Solved in [dc46037](https://github.com/yetanotherco/aligned_layer/pull/268/commits/dc460377b873c514d3fafe2f9009e4617ec3b09d)